tests/test.py

```diff
@@ -164,9 +164,16 @@ def test_get_signatureTimestamp(self):
         signatureTimestamp = self.yt.get_signatureTimestamp()
         self.assertIsNotNone(signatureTimestamp)
 
-    ###############
+    def test_set_tasteprofile(self):
+        self.assertRaises(self.yt.set_tasteprofile("Galactic"))
```

Comment:
The test should also showcase passing in multiple artists and also test with the authenticated profile `yt_auth`.